rend3-gltf/Cargo.toml

```diff
@@ -22,7 +22,7 @@ ddsfile = { version = "0.5", optional = true }
 float-ord = "0.3.2"
 glam = "0.21.2"
 gltf = { version = "1.0", default-features = false, features = ["KHR_lights_punctual", "KHR_texture_transform", "KHR_materials_unlit", "extras", "names", "utils"] }
-image = { version = "0.24", default-features = false }
```

Comment:
Unrelated change, but had issues loading models with embedded textures. And this seems to fix it.

Reply:
Not sure why this fixed it, but does it work if you keep this line and add a manual dependency on `image` in the user crate specifying the required features (like `"png"`).

AFAIK, rend3-gltf does not specify any features for its `image` dependencies, but some features of the image crate (like "png" or "jpg") are required by `gltf` if you want to load embedded textures, this is by design (so that you don't have to compile, e.g. a `tiff` parser if your game doesn't use that file format).

Reply:
Yeah, I thought it was a weird fix, it was also fixed by updating the image dependency from 0.23 to 0.24.